tests/unit/test_sandbox.py

```diff
@@ -158,9 +157,7 @@ def test_ssh_box_stateful_cmd_run_as_devin(temp_dir):
     ), patch.object(config, 'run_as_devin', new='true'), patch.object(
         config, 'sandbox_type', new='ssh'
     ):
-        for box in [
-            DockerSSHBox()
-        ]:  # FIXME: DockerExecBox() does not work with stateful commands
+        for box in [DockerSSHBox()]:
```

Comment:
Same, we can simplify a bit this code that used to expect multiple boxes when it doesn't anymore 😅

Reply:
done! replaced all occurrences of `[DockerSSHBox()]` with assignment!